Unable to build examples bundled with Yampa-0.10.6.1

This is with GHC 8.2.1, if that's significant:

```
$ cabal get Yampa-0.10.6.1
Unpacking to Yampa-0.10.6.1/
$ cd Yampa-0.10.6.1/
$ cabal configure -fexamples
Resolving dependencies...
Configuring Yampa-0.10.6.1...
$ cabal build
Preprocessing library for Yampa-0.10.6.1..
Building library for Yampa-0.10.6.1..
<build output elided>
Preprocessing executable 'yampa-examples-sdl-bouncingbox' for Yampa-0.10.6.1..
Building executable 'yampa-examples-sdl-bouncingbox' for Yampa-0.10.6.1..
[1 of 1] Compiling Main             ( examples/yampa-game/MainBouncingBox.hs, dist/build/yampa-examples-sdl-bouncingbox/yampa-examples-sdl-bouncingbox-tmp/Main.o )

examples/yampa-game/MainBouncingBox.hs:8:1: error:
    Could not find module ‘YampaSDL’
    Use -v to see a list of the files searched for.
  |
8 | import YampaSDL
  | ^^^^^^^^^^^^^^^
```

--- a/Yampa.cabal
+++ b/Yampa.cabal
@@ -139,6 +139,7 @@ test-suite regression
 
 executable yampa-examples-sdl-bouncingbox
   main-is: MainBouncingBox.hs
+  other-modules: YampaSDL
   hs-source-dirs:  examples/yampa-game/
   ghc-options : -O3 -Wall -fno-warn-name-shadowing
   build-Depends: base < 5, random, deepseq, SDL, Yampa
@@ -147,6 +148,7 @@ executable yampa-examples-sdl-bouncingbox
 
 executable yampa-examples-sdl-circlingmouse
   main-is: MainCircleMouse.hs
+  other-modules: YampaSDL
   hs-source-dirs:  examples/yampa-game/
   ghc-options : -O3 -Wall -fno-warn-name-shadowing
   build-Depends: base < 5, random, deepseq, SDL, Yampa
@@ -155,6 +157,7 @@ executable yampa-examples-sdl-circlingmouse
 
 executable yampa-examples-sdl-wiimote
   main-is: MainWiimote.hs
+  other-modules: YampaSDL
   hs-source-dirs:  examples/yampa-game/
   ghc-options : -O3 -Wall -fno-warn-name-shadowing -rtsopts
   build-Depends: base < 5, random, deepseq, SDL, hcwiid, Yampa

